### ISSUE TITLE
The badge indicating number of notifications does not read out on a screen reader

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#602]
The badge indicating number of notifications over the "Home" button does not read out on a screen reader. It should read out the number of notifications i.e. "Home, Notifications, 2"

**Comment by amanji** [comment]
https://github.com/hyperledger/aries-mobile-agent-react-native/pull/470

**Comment by knguyenBC** [comment]
Checked build 586 both on iOS and Android and both screen readers read the notifications.
